src/routes/[...lang]/__layout.svelte

```diff
@@ -5,23 +5,35 @@
   import { page } from '$app/stores';
   registerAll();
   let defaultLang = 'en';
-  let localeRegex = new RegExp(`^/(${availableLocales.join('|')})/`)
+  let localeRegex = new RegExp(`^/(${availableLocales.join('|')})(/|$)`)
   export async function load({ page: { path } }) {
     const lang = (localeRegex.exec(path) || [null, defaultLang])[1];
     init({
       initialLocale: lang,
       fallbackLocale: defaultLang
     });
     await waitLocale()
-    return {};
+    return {
+      props: {
+        lang
+      }
+    };
   }
 </script>
 <script lang="ts">
+  export let lang: string
+  
+  const flags = import.meta.globEager('../../lib/flags/*.svg')
```

Comment:
Can you use the `$lib` alias here?

Reply:
Sadly no because vite does not handle module aliases in globs. They must be relative paths.

But we could move the flags to `src/routes/_flags` to have `import.meta.globEager('./_/flags/*.svg')`.